microsoft-365/enterprise/view-account-license-and-service-details-with-microsoft-365-powershell.md

```diff
@@ -93,16 +93,81 @@ This example shows the services that user BelindaN@litwareinc.com has access to
 (Get-MgUserLicenseDetail -UserId belindan@litwareinc.com -Property ServicePlans)[0].ServicePlans
 ```
 
-To view all the services for a user who has been assigned *multiple licenses*, use the following syntax:
+This script will generate a CSV file for all users with columns for `DisplayName`, `UserPrincipalName`, `SkuId`, `SkuName`, and each of the `ServicePlans`. If a user does not have a certain service plan assigned, it will show up as "Not Assigned" in the corresponding column. Please replace `"C:\Users\Public\M365LicenseReport.csv"` with your desired output path.
```

Comment:
```suggestion
This script will generate a CSV file for all users with columns for `DisplayName`, `UserPrincipalName`, `SkuId`, `SkuName`, and each of the `ServicePlans`. If a user does not have a certain service plan assigned, it will show up as "Not Assigned" in the corresponding column. You should replace `"C:\Users\Public\M365LicenseReport.csv"` with your desired output path.
```